specs/math.txt

```diff
@@ -0,0 +1,58 @@
+Run this with `cargo run -- -M -s specs/math.txt`.
+
+Mathematical expressions extension.
+See the following link to know the syntax.
+https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions
+
+Inline-level mathematical expressions:
+
+```````````````````````````````` example
+This sentence uses `$` delimiters to show math inline:  $\sqrt{3x-1}+(1+x)^2$
+$\sum_{k=1}^n a_k b_k$: Mathematical expression at head of line
+.
+<p>This sentence uses <code>$</code> delimiters to show math inline:  <span class="math">sqrt{3x-1}+(1+x)^2</span>
```

Comment:
These examples show the issue where the first character of inline math is dropped.